docs/user/security/audit-logging.asciidoc

```diff
@@ -248,6 +332,74 @@ Refer to the corresponding {es} logs for potential write errors.
 | `success` | User has accessed an alert as part of a search operation.
 | `failure` | User is not authorized to access alerts.
 
+.2+| `case_get`
+| `success` | User has accessed a case.
+| `failure` | User is not authorized to access a case.
+
+.2+| `case_resolve`
+| `success` | User has accessed a case.
+| `failure` | User is not authorized to access a case.
+
+.2+| `case_find`
+| `success` | User has accessed a case as part of a search operation.
+| `failure` | User is not authorized to search for cases.
+
+.2+| `case_ids_by_alert_id_get`
+| `success` | User has accessed cases as part of a search operation.
+| `failure` | User is not authorized to search for cases.
+
+.2+| `case_get_metrics`
+| `success` | User has accessed a case.
+| `failure` | User is not authorized to access a case.
+
+.2+| `cases_get_metrics`
+| `success` | User has accessed cases.
+| `failure` | User is not authorized to access cases.
+
+.2+| `case_configuration_find`
+| `success` | User has accessed a case configuration as part of a search operation.
+| `failure` | User is not authorized to search for case configurations.
+
+.2+| `case_comment_get_metrics`
+| `success` | User has accessed case comments.
+| `failure` | User is not authorized to access case comments.
+
+.2+| `case_comment_alerts_attach_to_case`
+| `success` | User has accessed case comments.
+| `failure` | User is not authorized to access case comments.
+
+.2+| `case_comment_get`
+| `success` | User has accessed a case comment.
+| `failure` | User is not authorized to access a case comment.
+
+.2+| `case_comment_get_all`
+| `success` | User has accessed case comments.
+| `failure` | User is not authorized to access case comments.
+
+.2+| `case_comment_find`
+| `success` | User has accessed a case comment as part of a search operation.
+| `failure` | User is not authorized to search for case comments.
+
+.2+| `case_tags_get`
+| `success` | User has accessed a case.
+| `failure` | User is not authorized to access a case.
+
+.2+| `case_reporters_get`
+| `success` | User has accessed a case.
+| `failure` | User is not authorized to access a case.
+
+.2+| `case_find_statuses`
+| `success` | User has accessed a case as part of a search operation.
+| `failure` | User is not authorized to search for cases.
+
+.2+| `case_user_actions_get`
+| `success` | User has accessed a case user action.
```

Comment:
> | `success` | User has accessed a case user action.

FYI The API docs currently refer to "case activity" and "user activity", which might be simpler to understand than "case user action".